src/test/java/seedu/address/ui/HelpWindowTest.java

```diff
@@ -24,10 +18,4 @@ public void setUp() throws Exception {
         FxToolkit.showStage();
         helpWindowHandle = new HelpWindowHandle(helpWindowStage);
     }
-
-    @Test
-    public void display() {
-        URL expectedHelpPage = HelpWindow.class.getResource(USERGUIDE_FILE_PATH);
-        assertEquals(expectedHelpPage, helpWindowHandle.getLoadedUrl());
-    }
```

Comment:
Hmm I don't think this test should be deleted. It is testing if the UserGuide.html can be displayed properly and has nothing to do with the BrowserPanel.